examples/tutorials/advanced/working_with_panel.py

```diff
@@ -0,0 +1,138 @@
+r"""
+Dynamic data visualization using ``Panel``
+==========================================
+*Please run the following code examples in a notebook environment otherwise
+the interactive parts of this tutorial will not work. You can use the button
+"Download Jupyter notebook: panel_extension.ipynb" at the bottom of this page
+to download this script as a Jupyter notebook.*
```

Comment:
```suggestion

.. note::

    Please run the following code examples in a notebook environment otherwise
    the interactive parts of this tutorial will not work. You can use the button
    "Download Jupyter notebook: panel_extension.ipynb" at the bottom of this page
    to download this script as a Jupyter notebook.*
```